requirements/conda.txt

```diff
@@ -11,8 +11,8 @@ pickleshare
 pyzmq
 chardet>=2.0.0
 numpydoc
-pyqt5
+pyqt>=5.6
 keyring
 spyder-kernels>=1.0
-python-language-server
+python-language-server=0.19*
```

Comment:
With conda, you can just do ``0.19``, although it might just ignore the asterisk (I know it does work when you do 0.19.*``.

Reply:
Right, it's simpler.